### ISSUE TITLE
Next Nuget Release

### ISSUE THREAD

**Issue by rog1039** [AutoMapper/AutoMapper#2900]
I am wondering when the next nuget release will occur?

I upgraded to V 8.0.0 and started getting runtime exceptions in my program. Downgrading to V 7.0.1 fixed the issue.

I also cloned the AutoMapper repository and ran my test case, and when I am on the latest commit, 9e2ebf94ec1b080dfdcd1d2270cba39c5255d810 from 11/23/2018, the test runs without issue. However, if I am on the commit tagged as v8.0.0, af60a492aba1d8d24ef807450b904cfa30282dee, then the test fails. 

After some trial and error it seems that commit bc44acecb9fb7edf5fa10a40cab1cbec9f3e3ca6 fixes the issue and my code longer crashes. So I am hoping that a V 8.0.1 will be released soon with the latest code. 

Any thoughts on when the nuget package might be updated?

Thanks,
Paul

**Comment by lbargaoanu** [comment]
A repro would help. Make a [gist](https://gist.github.com/lbargaoanu/9c7233441c3a3413cc2b9b9ebb5964a9) that we can execute and see fail.

**Comment by rog1039** [comment]
Here is the test case I have been using: https://gist.github.com/rog1039/db1ede2537c9e055ff605ca43d3d9995

I was just copying and pasting that into different projects. So it works fine in projects using V7 code or the latest source code, but doesn't work on V8.0.0 code from what I have seen.

**Comment by lbargaoanu** [comment]
That's way too complicated.

**Comment by rog1039** [comment]
I've stripped away almost everything and the test case still fails specifically on V8.0.0.

https://gist.github.com/rog1039/9f6f6d99f2ec97d48a2691c4b9b13656

And the stack trace where it fails:
```
System.InvalidOperationException: Sequence contains no elements
   at System.Linq.Enumerable.Last[TSource](IEnumerable`1 source)
   at AutoMapper.ConstructorParameterMap.get_SourceType()
   at AutoMapper.Execution.TypeMapPlanBuilder.<CheckForCycles>g__ResolveMemberTypeMap|17_4(IMemberMap memberMap)
   at AutoMapper.Execution.TypeMapPlanBuilder.<CheckForCycles>b__17_1(IMemberMap pm)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at AutoMapper.Execution.TypeMapPlanBuilder.CheckForCycles(HashSet`1 typeMapsPath)
   at AutoMapper.Execution.TypeMapPlanBuilder.CreateMapperLambda(HashSet`1 typeMapsPath)
   at AutoMapper.TypeMap.CreateMapperLambda(IConfigurationProvider configurationProvider, HashSet`1 typeMapsPath)
   at AutoMapper.TypeMap.Seal(IConfigurationProvider configurationProvider)
   at AutoMapper.MapperConfiguration.ResolveTypeMap(TypePair typePair, ITypeMapConfiguration inlineConfiguration)
   at AutoMapper.MapperConfiguration.ResolveTypeMap(TypePair typePair)
   at AutoMapper.Execution.TypeMapPlanBuilder.<CheckForCycles>g__ResolveTypeMap|17_5(TypePair types)
   at AutoMapper.Execution.TypeMapPlanBuilder.<CheckForCycles>g__ResolveMemberTypeMap|17_4(IMemberMap memberMap)
   at AutoMapper.Execution.TypeMapPlanBuilder.<CheckForCycles>b__17_1(IMemberMap pm)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at AutoMapper.Execution.TypeMapPlanBuilder.CheckForCycles(HashSet`1 typeMapsPath)
   at AutoMapper.Execution.TypeMapPlanBuilder.CreateMapperLambda(HashSet`1 typeMapsPath)
   at AutoMapper.TypeMap.CreateMapperLambda(IConfigurationProvider configurationProvider, HashSet`1 typeMapsPath)
   at AutoMapper.TypeMap.Seal(IConfigurationProvider configurationProvider)
   at AutoMapper.MapperConfiguration.Seal()
   at AutoMapper.MapperConfiguration..ctor(MapperConfigurationExpression configurationExpression)
   at AutoMapper.MapperConfiguration..ctor(Action`1 configure)
   at AutoMapper.UnitTests.HolderTests.MethodName() in C:\Users\paulrogero\source\repos\AutoMapperTests\AutoMapperTestV8\Class1.cs:line 20
```

So I assume the relevant line in the stack trace is 
```
  at AutoMapper.ConstructorParameterMap.get_SourceType()
```

given the commit where the test case starts passing again is bc44acecb9fb7edf5fa10a40cab1cbec9f3e3ca6 with a commit message of "a constructor parameter with no name won't match anything"

**Comment by lbargaoanu** [comment]
As a workaround, you can disable constructor mapping.

**Comment by lbargaoanu** [comment]
I guess we'll have to see how many people hit this.

**Comment by jbogard** [comment]
Let’s give it a week or so? See if there are any other major bugs. There’s
a lot of API changes so it might be longer to get enough people migrated.

**Comment by lbargaoanu** [comment]
Anyway, for this one, there seem to be a few ways to workaround it.

**Comment by replaysMike** [comment]
> I guess we'll have to see how many people hit this.

Count me in on this. Fails consistently on 8.0.0 and was annoying to find.

**Comment by Condor2708** [comment]
Same Error

**Comment by gpcaretti** [comment]
same error for me too

**Comment by Bosch571** [comment]
Got the same error as well.

**Comment by brown0617** [comment]
Same error

**Comment by eyesis** [comment]
I'm getting that error too, and it's fixed by 8.1.0-ci-01541. When will this be released?

@lbargaoanu says there is a simple workaround, but disabling constructor mapping doesn't fix the issue for me.

**Comment by lbargaoanu** [comment]
In all the cases I've seen, there are simple workarounds.
